cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,21 @@
+import time
+
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help_ = "Waits for the database to be available"

```

Comment:
It's not a mistake, but it's better to use 'help' instead of 'help_'. Django management commands have a 'help' attribute for providing a description of the command. Even though 'help' is a built-in function in Python, in the context of the class, it's not a problem to use it. This way, you are following the Django convention.